temp/userStories.adoc

```diff
@@ -0,0 +1,83 @@
+= Anforderungen allgemein
+
+[Note]
+====
+* Administratoren legen beliebig viele Diskussionsforen an
+* Gruppen sollen eigene Foren erstellen und verwalten können
+** diese dürfen nur von Mitgliedern der Gruppe lesbar sein (außer Admins)
+* es gibt anonyme Posts, sonst werden Klarnamen verwendet
+* Foren sollen einen in einen Moderationsmodus versetzt werden können
+* es soll Möglichkeiten geben differenziert in gesamten Foren und Teilen dessen zu suchen
+====
+
+== User Stories
+
+=== Student, Standard
+
+1. Student meldet sich in MOPS an.
+2. Student klickt auf ein `berechtigtes` Forum.
+3. Der Student `erstellt` einen Beitrag in einem der Topics.
+4. Der Student `bearbeitet` seinen Beitrag, um weiter Informationen einzupflegen.
+5. Der Student meldet sich später wieder an, um die Antworten zu `lesen`.
+
+=== Student, Suche/anonym
+
+1. Student `sucht` nach einem alten Beitrag im `gesamten` Forum.
+2. Er findet den gesuchten Beitrag.
+3. Er möchte eine ihm unangenehme Frage zum Beitrag `anonym` im Forum stellen.
+4. Der Student entscheidet die Frage wieder zu `entfernen`.
+
+=== Student, Multimedia
+
+1. Student möchte einen Beitrag `mit einem Medium erstellen`.
+2. Das Medium ist größer als `10 MB` und die Erstellung des Beitrags wird vom Forum `abgelehnt`.
+3. Der Student fügt ein Medium `kleiner als 10 MB` hinzu und kann den Beitrag `erstellen`.
+4. Der Student versucht `erfolglos`, das Medium des Beitrags gegen eines `größer als 10 MB` zu `ersetzen`.
+5. Der Student `ersetzt` das Medium gegen ein anderes gültiges Medium.
+
+=== Student, Gruppe
+
+1. Eine Studentin `erstellt` ein eigenes `Gruppenforum` und wird `automatisch` zum Admin des Forums.
+2. Sie `fügt` weitere Studenten zum Forum `hinzu`.
+3. Sie `ernennt` eine andere Studentin zum Admin des Forums.
+4. Sie `schließt` einen anderen Studenten aus ihrem Forum `aus`.
+5. Sie `löscht` das Gruppenforum.
+
+=== Moderator, Freigabe
+
+1. Der Moderator erhält einen Beitrag aus einem moderierten Forum, den er `freigeben` soll.
+2. Der Moderator `liest` den Beitrag und `verändert` Teile des Inhalts.
```

Comment:
Important: If the Moderator is able to edit the content, it HAS to be displayed. No changes without the information that the content has changed. (Transparency)